cockroachdb/tests/test_check.py

```diff
@@ -18,6 +21,19 @@ def test_metrics(aggregator, instance, dd_run_check):
     assert_metrics(aggregator)
```

Comment:
Small nit, but could you also add 
```
    aggregator.assert_service_check('cockroachdb.openmetrics.health', ServiceCheck.OK)
    aggregator.assert_metrics_using_metadata(get_metadata_metrics(), check_submission_type=True)
```
to this test as well to match `test_security_metrics`?